test/src/api/commands/testSendKeys.js

```diff
@@ -22,7 +22,7 @@ describe('sendKeys', function() {
     MockServer.addMock({
       url: '/wd/hub/session/1352110219202/element/0/value',
       method: 'POST',
-      postdata: {value: ['p', 'a', 's', 's', 'w', 'o', 'r', 'd']},
+      postdata: {'text': 'password', 'value': ['p', 'a', 's', 's', 'w', 'o', 'r', 'd']},
```

Comment:
is this change related to mobile integration?

Reply:
No, this has nothing to do with the `mobile` update. But @swrdfish and I ran across a problem while writing a test-case for `click` command changes. So our written test-case was always passing even after receiving a 404 in response from selenium webdriver's `/execute/sync` request. It is signalling that the `click` had never occurred, which means our test case should fail ideally but it passed. Then after a lot of debugging we reached to this solution:

_lib/http/response.js_
```js
// for error we have to send empty response, otherwise selenium-webdriver treats the request as success.
this.responseData = (this.res.statusCode >= 400) ? '' : {};
```

After this change lots of test cases have been broken, and when we took a deeper look, we discovered those cases should fail (except it's expected) because they are also receiving `4XX` in response. So we have fixed them all.